app/src/main/java/fr/neamar/kiss/result/Result.java

```diff
@@ -76,16 +77,14 @@ else if (pojo instanceof ShortcutsPojo)
      *
      * @param text Text to highlight
      * @param positions List of matched positions
-     * @param context Application context
      * @return Spannable displayable on a TextView
      */
-    static Spanned enrichText(String text, List<Pair<Integer, Integer>> positions, Context context) {
+    static Spanned enrichText(String text, List<Pair<Integer, Integer>> positions) {
         SpannableString enriched = new SpannableString(text);
 
-        int primaryColor = UIColors.getPrimaryColor(context);
         for (Pair<Integer, Integer> position : positions) {
             enriched.setSpan(
-                    new ForegroundColorSpan(primaryColor),
+                    CharacterStyle.wrap(new ForegroundColorSpan(UIColors.getPrimaryColor())),
```

Comment:
Wait, you're now creating items twice for no reasons :p

Reply:
oh shit